tests/containers/kubeadm.pm

```diff
@@ -0,0 +1,46 @@
+# SUSE's openQA tests
+#
+# Copyright 2023 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Package: kubernetes-*-kubeadm kubernetes*-client
+# Summary: Test kubeadm to bootstrap single node k8s cluster.
+# Maintainer: QE-C team <qa-c@suse.de>
+
+use strict;
+use warnings;
+use base "consoletest";
+use testapi;
+use utils qw(systemctl zypper_call);
+
+sub run {
+    select_console("root-console");
+    zypper_call "in patterns-kubernetes-kubeadm conntrackd iptables";
+    systemctl "start containerd";
+
+    record_info 'Test #1', 'Test: Initialize kubeadm';
+    my $kube_version = script_output("kubelet --version |sed -e 's/Kubernetes v//g'");
+    # Need to skip kube-proxy because of https://github.com/kubernetes/kubeadm/issues/2699
+    assert_script_run("kubeadm init --v=5 --skip-phases=addon/kube-proxy --kubernetes-version=$kube_version | tee /dev/$serialdev", 300);
+
+    record_info 'Test #2', 'Test: Configure kubectl';
+    assert_script_run('mkdir -p ~/.kube');
+    assert_script_run('cp -i /etc/kubernetes/admin.conf ~/.kube/config');
+
+    sleep 60;
```

Comment:
This can be improved. Consider `kubectl wait` command.

Reply:
Shouldn't this `wait` run after `apply` ?

Reply:
I couldn't find a way to make `wait` work before `apply`.

Reply:
What about something like
```bash
kubectl wait --for=condition=Ready nodes --all
```

Reply:
> What about something like
> 
> ```shell
> kubectl wait --for=condition=Ready nodes --all
> ```

Will have to wait until  https://bugzilla.opensuse.org/show_bug.cgi?id=1218540 is fixed.